.github/workflows/binary.yml

```diff
@@ -7,17 +7,20 @@ on:
   schedule:
     # Run every morning to detect flakiness and broken dependencies
     - cron: '17 8 * * *'
+env:
+  ROS_DISTRO: galactic
+  BRANCH: master
 
 jobs:
   ci_binary:
     name: Galactic binary job
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        env:
-          - {ROS_DISTRO: galactic, ROS_REPO: main}
-          - {ROS_DISTRO: galactic, ROS_REPO: testing}
+          ROS_REPO: [main, testing]
```

Comment:
can we have this on a separate workflow called "ABI Compatibility" or "ABI Check" and have the tests disabled?